lib/presto/client/faraday_client.rb

```diff
@@ -60,7 +60,9 @@ def self.faraday_client(options)
       if options[:user] && options[:password]
         faraday.basic_auth(options[:user], options[:password])
       end
-
+      if options[:follow_redirect] == true
```

Comment:
```suggestion
      if options[:follow_redirect]
```
Usually in Ruby it doesn't need `== true`, unless you explicitly want to compare with true.

Reply:
Thank you!